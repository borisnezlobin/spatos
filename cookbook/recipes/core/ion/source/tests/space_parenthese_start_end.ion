let word = "hello"
let list = [list hello]

# exepcted: 5 
let length = $len( $word  );
echo $length

# exepcted: 5 
let length = $len( $word);
echo $length

# exepcted: 5 
let length = $len($word  );
echo $length

# exepcted: 5 
let length = $len(   ${word}  );
echo $length

# exepcted: 5 
let length = $len( ${word});
echo $length

# exepcted: 
# h 
# llo 
for char in @split(    ${word}    'e' );
  echo $char
end

# exepcted: 5 
let length = $len(${word}  );
echo $length

# exepcted: 2
let length = $len(  @{list}  );
echo $length

# exepcted: 2
let length = $len(   @list  );
echo $length

# exepcted: 3
let length = $len(            "xxx");
echo $length

# exepcted: 6 
let length = $len(    [2 4 5 89 789 3245]  );
echo $length

# exepcted: 6 
let length = $len([2 4 5 89 789 3245]  );
echo $length

