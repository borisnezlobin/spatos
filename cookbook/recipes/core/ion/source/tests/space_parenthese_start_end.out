5
5
5
5
5
h
llo
5
2
2
3
6
6
